kubernetes-overview.el

```diff
@@ -374,24 +374,27 @@
       (add-hook 'kill-buffer-hook (kubernetes-utils-make-cleanup-fn buf) nil t))
     buf))
 
-(defun kubernetes-overview-set-sections (sections)
-  "Set which sections are displayed in the overview.
+(defun kubernetes-overview-select-views (&rest sections)
+  "Set which views are displayed in the overview (by multiselect).
 
 SECTIONS is a list of sections to display.  See
 `kubernetes-overview-custom-views-alist' and
 `kubernetes-overview-views-alist' for possible values."
   (interactive
    (let* ((views (append kubernetes-overview-custom-views-alist kubernetes-overview-views-alist))
           (names (-uniq (--map (symbol-name (car it)) views)))
-          (choice (intern (completing-read "Overview view: " names nil t))))
-     (list (alist-get choice views))))
+          (choices (completing-read-multiple "Overview view: " names nil t))
+          (list (alist-get (intern (car choices)) views)))
+     (dolist (choice (cdr choices))
+       (setq list  (append list (cdr (alist-get (intern choice) views)))))
+     (delete-dups list)))
```

Comment:
Given that `list` is a reserved keyword in Elisp let's not name the variable `list` here.

Regarding your testability point, the dedupe logic here I can see us moving out into a function like `kubernetes-overview-views--dedupe` that we then call here. That would make the dedupe logic testable. I recommend this approach.